lib/laa_crime_schemas/structs/applicant.rb

```diff
@@ -4,7 +4,7 @@ module LaaCrimeSchemas
   module Structs
     class Applicant < Person
       attribute :date_of_birth, Types::JSON::Date
-      attribute :nino, Types::String
+      attribute? :nino, Types::String
```

Comment:
It needs to be `attribute :nino, Types::String.optional`, as what we want, is presence of the `nino` attribute (as in the schema is marked as `required`), but optionality in its content (thus allowing `null`).